### DISCUSSION TITLE
Add link to delete fork of merged pull request

### DISCUSSION THREAD

**Discussion by janosh** [community/community#49625] [Mobile]
The web UI offers to delete either the branch or the whole fork of a repo whose PR was merged into upstream. Would be nice if the app also offered a link to delete the repo. Currently there's just a button to delete the branch.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.103.0 (93212509)
Device: iPhone15,2 (iOS 16.3)
</details>

**Comment by aconsuegra** [comment]
Thanks for getting in touch and for the feature request @janosh! I'll forward this feedback to our product team for evaluation, stay tuned!
